yarn.lock

```diff
@@ -309,9 +309,9 @@
     strip-json-comments "^3.1.1"
 
 "@fiatconnect/fiatconnect-types@^3.0.0":
-  version "3.0.0"
-  resolved "https://registry.yarnpkg.com/@fiatconnect/fiatconnect-types/-/fiatconnect-types-3.0.0.tgz#684b8f5c6beefb98db3a2a3c41066b976e099269"
-  integrity sha512-JMyAgkHNufqbo9roDmUpzhZH3UMbhyuLYYMI9kE1qQTtQMPVlPz9zHbS2syFz2Ymdjl7wzgIijph+QckYwwP8A==
+  version "3.1.0"
```

Comment:
Should we also upgrade package.json? Any consumer of this package should use the latest, but this will still allow them to use 3.0.0, which could break the sdk if it uses any 3.1.0 features.

Reply:
that is a good point

Reply:
I updated it